client/src/components/framework/toasters.js

```diff
@@ -1,31 +1,33 @@
 import { Position, Toaster, Intent } from "@blueprintjs/core";
 
 /** Singleton toaster instance. Create separate instances for different options. */
-
-const ToastTopCenter = Toaster.create({
-  className: "recipe-toaster",
-  position: Position.TOP,
-  maxToasts: 4,
-});
+let ToastTopCenter = null;
```

Comment:
is this still needed now we have JSDom? Thanks!

Reply:
I dropped JSDom on the unit tests since the tests themselves don't need any browser API's